docs/UserGuide.md

```diff
@@ -1,226 +1,23 @@
-
---------------------------------------------------------------------------------------------------------------------
-
-## Features
-
-<div markdown="block" class="alert alert-info">
-
-**:information_source: Notes about the command format:**<br>
-
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
-
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
-
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
-  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
-
-</div>
-
-### Instantaneous launching
-Users that have Java 11 or above installed in their computers can launch the Duke The Market program 
-by double clicking on the file.
-
-### Saving the data
-Duke The Market data are saved in the hard disk automatically after any command that changes the data. 
-There is no need to save manually.
-
-### Viewing help : `help`
-
-Shows a message explaning how to access the help page.
-
-![help message](images/helpMessage.png)
-
-Format: `help`
-
-### Adding a contact: `add`
-
-Adds a contact to the address book.
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [g/GENDER] [d/DOB] [ph/NEW_PURCHASE_DETAIL] [t/TAG]`
-
-- The compulsory parameters are: name (`n`), phone number (`p`), email (`e`), address (`a`)
-- The parameters in [ ] are optional parameters, including gender (`g`), date of birth (`d`), purchase history (`ph`), tag (`t`).
-- A person in the contact list can have more than 1 tag.
-
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 g/m d/20 MAR 2002
-  ph/[12.04.2022, 101.25, 3, Orange, Umbrella, Soap]`
-* `add n/Betsy Crowe e/betsycrowe@example.com a/Newgate Prison p/1234567 t/friend t/criminal g/f g/14.12.1998
-  ph/[13.05.2021 12.25, 3, Sweets, Banana, Battery]`
-
-__Optional Parameter 1: Gender__
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [g/GENDER]`
-
-- Adds gender to a person in the contact list. The genders accepted by the contact list are: `M`/`m` for male,  `F`/`f` for female.
+- Adds date of birth to a person in the contact list. Date formats accepted are: d.M.yyyy , dd MMM yyyy, M d, yyyy
```

Comment:
TBH it took me a while to understand the date formats, do you think dd.mm.yyyy would better represent 20.03.2000?
Also, month dd, yyyy to represent March 20, 2000 & keeping the current dd MMM yyyy for dd MMM yyyy? What do ya think?

Reply:
mmmm sounds gd! will change